packages/zowe-explorer-api/package.json

```diff
@@ -27,6 +27,7 @@
     "@zowe/zos-tso-for-zowe-sdk": "8.0.0-next.202403132009",
     "@zowe/zos-uss-for-zowe-sdk": "8.0.0-next.202403132009",
     "@zowe/zosmf-for-zowe-sdk": "8.0.0-next.202403132009",
+    "lodash.isequal": "^4.5.0",
```

Comment:
Good call - I looked into this a bit more and it turns out we've had this in ZE for some time, not sure what I can do about that though for this PR because it stretches into both the edit attributes and edit history webviews - for the sake of scope creep we might want to save this for a future PR.

But, since this PR added it to ZE API, I've removed the reference in `BaseProvider`, replaced it with a similar implementation named `areContentsEqual` and removed `lodash.isequal` from the API `package.json`.